client_lib/test/unit/client/commands/repo/test_cudl.py

```diff
@@ -517,6 +517,33 @@ def test_summary(self):
         self.assertEqual(self.recorder.lines[0], 'abcdef  ABCDEF\n')
         self.assertEqual(self.recorder.lines[1], 'xyz     XYZ\n')
 
+    def test_summary_empty_display_name(self):
```

Comment:
It will be good to add a docstring which will describe what this test does, like [here](https://github.com/pulp/pulp/blob/master/server/test/unit/server/controllers/test_importer.py), for example.